dpdata/vasp/outcar.py

```diff
@@ -66,19 +76,33 @@ def get_frames (fname, begin = 0, step = 1) :
     cc = 0
     while len(blk) > 0 :
         if cc >= begin and (cc - begin) % step == 0 :
-            coord, cell, energy, force, virial, is_converge = analyze_block(blk, ntot, nelm)
-            if is_converge : 
+            if ml == False:
+                coord, cell, energy, force, virial, is_converge = analyze_block(blk, ntot, nelm)
+                if is_converge : 
+                    if len(coord) == 0:
+                        break
+                    all_coords.append(coord)
+                    all_cells.append(cell)
+                    all_energies.append(energy)
+                    all_forces.append(force)
+                    if virial is not None :
+                        all_virials.append(virial)
+            else:
+                coord, cell, energy, force, virial = analyze_block_ml(blk, ntot)
                 if len(coord) == 0:
-                    break
+                        break
                 all_coords.append(coord)
                 all_cells.append(cell)
                 all_energies.append(energy)
                 all_forces.append(force)
                 if virial is not None :
                     all_virials.append(virial)
-        blk = get_outcar_block(fp)
-        cc += 1
-        
+        if ml == False:
+            blk = get_outcar_block(fp)
+        else:
+            blk = get_outcar_block_ml(fp)
+        cc+=1
```

Comment:
This line does not obey [PEP-8](https://peps.python.org/pep-0008/). (The original code is correct)